src/components/Home/Home.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Redirect, useHistory } from 'react-router-dom';
+import getToken from '../../lib/tokens';
+
+const Home = () => {
+  const userToken = localStorage.getItem('token');
+
+  let history = useHistory();
```

Comment:
`useHistory` hook. nice! 

Nit: if you want to, you can declare this as a `const` rather than a `let`. Mutating `const`'s is fine with things like `.push`, you just can't reassign. Not a big deal though.